resources/model/OpenStudio.idd

```diff
@@ -40723,7 +40723,6 @@ OS:ExternalInterface:FunctionalMockupUnitExport:To:Schedule,
        \retaincase
   N1 ; \field Initial Value
        \type real
-       \required-field
```

Comment:
I'm not 100% sure this is correct on E+ side, asking for clarification in 

* https://github.com/NREL/EnergyPlus/pull/10943#discussion_r2444489307